src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,10 @@
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  border: 0;
+  clip: rect(0 0 0 0);
+}
```

Comment:
do you need it?

Reply:
> 

for card title  i used tag h2,for best semantic in html i create tag h1 and hide it)